go-selinux/selinux_linux.go

```diff
@@ -334,36 +311,36 @@ func FileLabel(fpath string) (string, error) {
 }
 
 /*
-SetFSCreateLabel tells kernel the label to create all file system objects
+setFSCreateLabel tells kernel the label to create all file system objects
 created by this task. Setting label="" to return to default.
 */
-func SetFSCreateLabel(label string) error {
+func setFSCreateLabel(label string) error {
 	return writeAttr("fscreate", label)
 }
 
 /*
-FSCreateLabel returns the default label the kernel which the kernel is using
+fsCreateLabel returns the default label the kernel which the kernel is using
 for file system objects created by this task. "" indicates default.
```

Comment:
I kept the goDoc comments in these implementations, but we could remove them if we don't think they're useful here (as they're repeating what's in the exported versions

Reply:
I think it's OK to leave it as it is